machine-learning/app/config.py

```diff
@@ -13,7 +13,7 @@ class Settings(BaseSettings):
     facial_recognition_model: str = "buffalo_l"
     min_tag_score: float = 0.9
     eager_startup: bool = True
-    model_ttl: int = 300
+    model_ttl: int = 0
```

Comment:
Why is the cache ttl changed to 0?

Reply:
Ah, no unloading because it doesn't work great ATM.

Reply:
Yeah I don't want to risk exacerbating memory usage for now. We can re-enable it by default once it's better.